sdk/search/Azure.Search.Documents/src/autorest.md

```diff
@@ -65,6 +54,17 @@ directive:
     $.additionalProperties = true;
 ```
 
+### Rename one of SearchError definitions
+
+SearchError is duplicated between two swaggers, rename one of them
+
+``` yaml
```

Comment:
cc @tg-msft @heaths

Reply:
We'll talk with @brjohnstmsft when he gets back about combining these swaggers or at least refactoring out common models. For now, though, since querying is the main focus, I'd keep searchindex's `SearchError` as is and rename searchservice's `SearchError` as `SearchServiceError`.